packages/components/src/Combobox/components/Trigger/TriggerChip.css

```diff
@@ -1,17 +1,18 @@
 .trigger {
   display: inline-flex;
-  gap: var(--space-smaller);
-  padding: var(--space-small);
-  border: var(--border-base) dashed var(--color-interactive);
+  gap: var(--space-small);
+  padding: var(--space-small) var(--space-base);
+  border: var(--border-base) solid rgb(230, 234, 235);
   border-radius: calc(var(--radius-large) * 5);
+  background-color: var(--color-surface);
```

Comment:
Doing this because I've switched TriggerChip to be a `<button>` for now (better semantically and for testing purposes) and that forces the background to be grey.  Once Chip is refactored we can look to sub that into this component